index.html

```diff
@@ -28,6 +28,68 @@ <h1>Hector<br>Pereira</h1>
                 <span>Start collaboration</span>
             </button>
         </section>
+        <section class="projects">
+            <h2>Projects</h2>
+            <div class="project-box">
+                <img src="img/project-1.png" alt="project-1" class="project-img">
+                <div class="project-details">
+                    <h3>Project names go here</h3>
+                    <div class="built">
+                        <span>HTML/CSS</span>
+                        <span>Ruby on Rials</span>
+                        <span>JavaScript</span>
+                    </div>
+                    <button>See this project →</button>
+                </div>
+            </div>
+            <div class="project-box">
+                <img src="img/project-2.png" alt="project-1" class="project-img">
+                <div class="project-details">
+                    <h3>Project names go here</h3>
+                    <div class="built">
+                        <span>HTML/CSS</span>
+                        <span>Ruby on Rials</span>
+                        <span>JavaScript</span>
+                    </div>
+                    <button>See this project →</button>
+                </div>
+            </div>
+            <div class="project-box">
+                <img src="img/project-3.png" alt="project-1" class="project-img">
+                <div class="project-details">
+                    <h3>Project names go here</h3>
+                    <div class="built">
+                        <span>HTML/CSS</span>
+                        <span>Ruby on Rials</span>
+                        <span>JavaScript</span>
+                    </div>
+                    <button>See this project →</button>
+                </div>
+            </div>
+            <div class="project-box">
+                <img src="img/project-4.png" alt="project-1" class="project-img">
+                <div class="project-details">
+                    <h3>Project names go here</h3>
+                    <div class="built">
+                        <span>HTML/CSS</span>
+                        <span>Ruby on Rials</span>
+                        <span>JavaScript</span>
+                    </div>
+                    <button>See this project →</button>
+                </div>
+            </div>
+        </section>
+        <section class="about">
+            <h2>About</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+            nisi ut aliquip ea eiusmodz</p>
+            <button>
+                <span>Get my CV</span>
+                <img src="img/download.png" alt="download" width="25">
+            </button>
```

Comment:
- [ ] You missed the technologies categories, please add it.

![image](https://user-images.githubusercontent.com/60631456/120720716-2e8a0a80-c492-11eb-8904-18d6b76a9241.png)

Reply:
:+1: Issue addressed successfully :heavy_check_mark: